src/test/java/seedu/address/testutil/TestUtil.java

```diff
@@ -302,7 +302,7 @@ public static Object getLastElement(List<?> list) {
      * @return
      */
     public static TestTask[] replaceTaskFromList(TestTask[] Tasks, TestTask Task, int index) {
-        Tasks[index] = Task;
+        Tasks[index - 1] = Task;
```

Comment:
The index we get doesn't include zero, that's why when we replace the task we need to minus one